bundle/edu.gemini.sp.vcs.reg/src/main/scala/edu/gemini/sp/vcs/reg/VcsRegistrar.scala

```diff
@@ -7,7 +7,7 @@ import edu.gemini.spModel.core.Peer
  * A service that provides persistent/mutable mapping between program ids and
  * VCS server locations.
  */
-trait VcsRegistrar {
+trait VcsRegistrar extends scala.collection.mutable.Publisher[VcsRegistrationEvent] {
```

Comment:
I believe the registration bundle should just need the corresponding `Subscriber` class.  I did this to replace the previous version which had a simple but custom publish/subscribe method and it seems to work fine with an OT subscriber and the registration bundle.